packages/block-library/src/gallery/editor.scss

```diff
@@ -75,8 +75,16 @@ figure.wp-block-gallery {
 		margin-top: -9px;
 		margin-left: -9px;
 	}
-}
 
+	// Temporary notice to display warning about mobile app compat. Can be removed for 6.0
+	.components-notice {
```

Comment:
Can this CSS rule be removed now? It doesn't look like it's targeting the snackbar notice.